### ISSUE TITLE
CJ4 - Navaids not displayed

### ISSUE THREAD

**Issue by TomL7753** [Working-Title-MSFS-Mods/fspackages#687]
Routing: LOWS15 DETS2B DETSA Y106 CHI Y664 IBRID IBRI1Q LIRQ05 

I tuned most of the navaids along the route with the following results:

    SI (410) NDB: only shows when BRG SRC is set to ADF1, not showing on ADF2
    TRE (302) NDB: didn't show at all
    115.75 FOL VOR: didn't show, no matter if BRG SRC was set to VOR1 or VOR2
    113.60 CEV VOR: the same
    108.80 RMN VOR: the same

Other NDB's/VOR's along the route were showing as expected. That corresponds with my earlier observations regarding 115.80 ALF VOR/DME (#650) not showing when tuned in for the ILS at EDDH. On a pre-patch flight from EGLC SND (363) NDB didn't display. Both are definitely in the MSFS database.

**Comment by TomL7753** [comment]
I have since found out that the issues seem to be related to the stock CJ4 and are as follows:

The avionics in the CJ4 does not display bearing information from NDBs that have a 0,5khz decimal frequency, e.g. Southend 363SND near London. The frequency is 362.5 khz, although the charts show 363. In real life, the ADF would tune to the station if you'd put in 363 (possibly even 362), but MSFS does not reflect that behaviour. And as you can not tune decimals for the ADF in the CJ4, NDB navigation (e.g. for the missed approach to rwy15 in LOWS) is hit and miss. TRE (302) has a frequency of 301.5 khz btw.

The "missing" VOR's are TACAN type and therefore not showing a bearing (which is correct) but should display the identifier and DME information afaik.

The 115.80ALF is a DME only station (decommissioned former VORDME). The information (identifier and distance) is shown on the G1000 in the stock C172 but not in the CJ4.

**Comment by mattnischan** [comment]
@TomL7753 -
As mentioned on the Avsim thread (and repeating here for posterity), we can't fix the sim's inability to tune to fractional frequencies for NDBs. In the sim database SND has a frequency of 362 in the sim, for example.

However, PR #803 will fix the DME only stations problem, so you should see that fix in the next release of the CJ4.

**Comment by mattnischan** [comment]
@TomL7753 -
Thanks for pointing out that the world map had the correct frequency. The game does allow it to be tuned, even if the navdata loader itself for things like the DATABASE page only returns whole numbers.

I have added 0.1khz spacing in PR #803 now which reflects the actual unit. I have confirmed that allows those NDBs to be tuned and have also adjusted the bearing pointer display to reflect this change also.
